docs/components/arm/_index.md

```diff
@@ -60,7 +60,9 @@ For configuration information, click on one of the supported arm models:
 | [`xArmLite`](xarmlite/) | [UFACTORY Lite 6](https://www.ufactory.cc/product-page/ufactory-lite-6) |
 | [`ur5e`](ur5e/) | [Universal Robots UR5e](https://www.universal-robots.com/products/ur5-robot) |
 
-Follow [this guide](/extend/modular-resources/examples/custom-arm/) to implement your custom arm as a [modular resource](/extend/modular-resources/).
+{{<modular-resources api="rdk:component:arm" type="arm">}}
```

Comment:
Nice, this is a really smart way of do this! (setting parameters for page-specific values on-page)